### PR TITLE
sale_quotation_number: Migration to v10.0

### PR DESCRIPTION
**Pull request by DarkoNikolovski**

### CODE REVIEW

File: sale_quotation_number/model/sale_order.py [hunk 1]
```diff
@@ -40,13 +41,12 @@ def create(self, vals):
...
-                    'origin': quo,
-                    'name': self.env['ir.sequence'].next_by_code(
-                        'sale.order')
-                })
-        return True
+    def action_confirm(self):
+        for sale in self:
+            quo = sale.name
+            sale.write({
+                'origin': quo,
```
Comment (paulius-sladkevicius) [inline sale_quotation_number/model/sale_order.py]: I suggest to put sale.name here instead creating quo.

**Review by paulius-sladkevicius — CHANGES_REQUESTED** [review]

**Comment by DarkoNikolovski** [comment]
Since there is no any additional notes for this PR I will create PR to OCA and will deploy it to the dev server for testing

**Comment by kkarolis** [comment]
@DarkoNikolovski is this still active, maybe we can close it?

**Comment by DarkoNikolovski** [comment]
I was waiting for merging on OCA before this to be merged. I can see only one suggestion there, will update the code and after that will merge it and close.

**Comment by paulius-sladkevicius** [comment]
@DarkoNikolovski, please check if it's still relevant, otherwise close.

**Pull request closed without merging**